src/Raven.Server/ServerWide/ClusterCommandsVersionManager.cs

```diff
@@ -45,6 +45,7 @@ public static class ClusterCommandsVersionManager
             [nameof(DeleteOngoingTaskCommand)] = Base40CommandsVersion,
             [nameof(DeleteValueCommand)] = Base40CommandsVersion,
             [nameof(EditExpirationCommand)] = Base40CommandsVersion,
+            [nameof(EditArchivalCommand)] = Base40CommandsVersion,
```

Comment:
This is not correct. Please put it at the bottom of the list and assign the new version number (54_003? - note that we have an open PR where we bump it: https://github.com/ravendb/ravendb/pull/15693)

Reply:
changed in the latest commit